app/admin/allocation_chart.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+ActiveAdmin.register_page 'Allocation Chart' do
+  menu label: proc { I18n.t('allocation_chart') }, parent: User.model_name.human(count: 2)
+
+  content title: I18n.t('allocation_chart') do
+    panel I18n.t('allocation_chart') do
+      allocations = AllocationsAndUnalocatedUsersQuery.new(Allocation, current_user.company).call
+      paginated_collection(allocations.page(params[:page]), download_links: false) do
+        table_for collection do
+          column(I18n.t('name'), :user)
```

Comment:
This column should also link to the engineer show page, right?

Reply:
Passing an ActiveRecord instance to `column` method automatically creates a link. Added a screenshot.